index.html

```diff
@@ -6913,33 +7054,70 @@ <h2><code>application/td+json</code> Media Type Registration</h2>
         </p>
       </dd>
       <dt>Published specification:</dt>
-      <dd><a href="https://w3c.github.io/wot-thing-description/">https://w3c.github.io/wot-thing-description/</a></dd>
+      <dd>
+        <a href="https://www.w3.org/TR/wot-thing-description11/">W3C 
+        Web of Things (WoT) Thing Description 1.1: 
+        https://www.w3.org/TR/wot-thing-description11/</a>
+        <p class="ednote" title="Versions">
+	This has been updated to point to the TD 1.1 specification.
+	Should we note that versions (and TDs vs. TMs) can be distinguished using
+	information interior to the content?
+	</p>
+        </dd>
       <dt>Applications that use this media type:</dt>
       <dd>
         All participating entities in the W3C Web of Things, that is,
         <a>Things</a>, <a>Consumers</a>, and Intermediaries as defined in the
-        <a href="https://www.w3.org/TR/wot-architecture/">Web of Things (WoT) Architecture</a>.</dd>
+        <a href="https://www.w3.org/TR/wot-architecture11/">W3C 
+	Web of Things (WoT) Architecture 1.1:
+        https://www.w3.org/TR/wot-architecture11/ 
+	</a> document.
+      </dd>
       <dt>Fragment identifier considerations:</dt>
-      <dd>See <a href="https://tools.ietf.org/html/rfc6839#section-3.1">RFC&nbsp;6839, section 3.1</a>.</dd>
+      <dd>See <a href="https://tools.ietf.org/html/rfc6839#section-3.1">RFC&nbsp;6839, section 3.1</a>.
+      <p class="ednote" title="Other Fragment Identifier Considerations">
+      We may 
+      want to explictly allow use of JSON Pointer as a fragment identifier [[RFC6901]].
+      There is in fact an assertion requiring them in TMs for <code>tm:required</code>.
+      These are internal references, however, so we <em>might</em> not have to allow
+      them for external references.
+      By default
+      the json media type does not support a fragment identifier but td+json could.
+      </p>
+      </dd>
       <dt>Additional information:</dt>
       <dd>
         <dl>
           <dt>Magic number(s):</dt>
           <dd>Not Applicable</dd>
           <dt>File extension(s):</dt>
-          <dd>.jsontd</dd>
+          <dd>.jsontd
+	  <p class="ednote" title="Other Extensions">
+	  We have been using .td.jsonld and .tm.jsonld in testing.  
+	  Should we also define these formally? 
+	  If we are using the same media type for TMs, should we still have
+	  a distinct filename extension?  
+	  If so, should we (also?) define .jsontm for consistency?
```

Comment:
I think that sounds good.

Also allowing `.td.jsonld` and `.tmjsonld` can't hurt, I suppose?

Reply:
an IANA discussion, not a security one.  I added this note to follow up on later...

Reply:
will mark this resolved, another set of PRs are needed to resolve the ednotes I added here.